packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.0.0-alpha.9",
+  "version": "6.0.0-alpha.11",
```

Comment:
Was there any change impacting `x-license-pro` package and warranting a release?

Reply:
Ah, it's probably the `mui core` [version bump](https://github.com/mui/mui-x/commit/a493eb2bb2f358cce360dc2d5fdf011f866c52b7)? 🤔 
It is only a bump in patch version. In theory it should not force any users to have different package versions resolved, because of this mismatch...
What do you think?

Reply:
I assumed that lerna only bumps the version of the changed packages.
For the license package, there was only this change: https://github.com/mui/mui-x/pull/7011/files#diff-09f4b5272f6882385a722c3dd6d78fc3512474df9ebaadcd4713221ff0ce9ae0